### PR TITLE
Add toggle option for devtools

### PR DESCRIPTION
**Pull request by ragingwind**
I think It would be useful when devtool is needed frequency. :)

### CODE REVIEW

**Comment by sindresorhus** [comment]
Sure, but options needs a better name. Maybe `showDevTools`. Also need to fix the code style. See travis.

**Comment by ragingwind** [comment]
Fixed

File: index.js [hunk 1]
```diff
@@ -44,8 +44,18 @@ function uninstallDebugContextMenu(win) {
 	win.webContents.removeListener('devtools-closed', deactivateDebugContextMenu);
 }
 
-module.exports = () => {
+module.exports = opts => {
+	opts = opts || {
+		toggle: false
+	};
```
Comment (sindresorhus) [inline index.js]: This is moot. Remove.

**Comment by sindresorhus** [comment]
And the option needs to be documented.

File: index.js (hunk 1 above)
Comment (ragingwind) [inline index.js]: my bad. I didn't change the name of the option. just remove it? we need a default value for opts. like this
```
opts = opts || {
        showDevTools: false
};
```

**Comment by ragingwind** [comment]
I added desc to readme as below. how about it?
```
## Usage
```js
var app = require('app');

require('electron-debug')({
	showDevTools: true
});

let win;

app.on('ready', function () {
	win = new BrowserWindow({
		width: 800,
		height: 600
	});
});
```

### Options

#### showDevTools

If true, DevTools will be shown after the app has been started.
```

File: index.js (hunk 1 above)
Comment (sindresorhus) [inline index.js]: There's no need to have default value for `false` as it's undefined by default which coerces to false.

**Comment by sindresorhus** [comment]
:+1:

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by ragingwind** [comment]
I merged but commit logs wasn't clear. it would be better that make new PR?

**Pull request closed without merging**